### PR TITLE
Remove meaningless comments and normalise formatting

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Not yet complete.

### CODE REVIEW

File: tests/cases/whitespace/comments-block [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    Qux.
+
+
+   Quux*/
+echo 'quuz';
+
+---MESSAGES---
+6:1 Squiz.Commenting.BlockComment.NoEmptyLineBefore
+16:1 Squiz.Commenting.BlockComment.CloserSameLine
+16:1 Squiz.Commenting.BlockComment.NoEmptyLineAfter
```
Comment (thewilkybarkid) [inline tests/cases/whitespace/comments-block]: Shame these aren't fixed.

File: tests/cases/whitespace/comments-line [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+---FIXED---
+<?php
+
+declare(strict_types=1);
+
+// Foo
+// Bar
+echo 'baz';    // qux
+  // quux
```
Comment (thewilkybarkid) [inline tests/cases/whitespace/comments-line]: Would rather the extra spaces before these two were removed.

File: src/Libero/ruleset.xml [hunk 3]
```diff
@@ -23,6 +23,42 @@
...
+        </properties>
+    </rule>
+    <rule ref="SlevomatCodingStandard.Commenting.EmptyComment"/>
+    <rule ref="SlevomatCodingStandard.Commenting.ForbiddenAnnotations">
+        <properties>
+            <property name="forbiddenAnnotations" type="array">
+                <element value="@api"/>
+                <element value="@author"/>
+                <element value="@category"/>
+                <element value="@copyright"/>
```
Comment (thewilkybarkid) [inline src/Libero/ruleset.xml]: https://github.com/doctrine/coding-standard/blob/0247a0d440e196d93611bd0f3c561ee12bd82789/lib/Doctrine/ruleset.xml#L161 also has `@created`, but can't see references to that anywhere else.

File: src/Libero/ruleset.xml [hunk 4]
```diff
@@ -23,6 +23,42 @@
...
+                <element value="@license"/>
+                <element value="@package"/>
+                <element value="@since"/>
```
Comment (stephenwf) [inline src/Libero/ruleset.xml]: Does this usually sit next to `@deprecated` annotations?

Comment (thewilkybarkid) [reply src/Libero/ruleset.xml]: No, it's the version it was introduced in.

Comment (stephenwf) [reply src/Libero/ruleset.xml]: ah fair enough, not as useful

File: tests/cases/whitespace/comments-line (hunk 2 above)
Comment (thewilkybarkid) [reply tests/cases/whitespace/comments-line]: `Generic.WhiteSpace.ScopeIndent` can be configured to fix the latter.

Comment (thewilkybarkid) [reply tests/cases/whitespace/comments-line]: Setting the `exact` value to `true` fixes it, but creates problems elsewhere. For example, indentation of array elements in removed. There is `Generic.Arrays.ArrayIndent` which can be used, but they seem to conflict.

Comment (thewilkybarkid) [reply tests/cases/whitespace/comments-line]: Looks problematic to try and resolve, let's not hold up the rest.

**Pull request merged**